# Datos de Demina
# color_detection:
#   ros__parameters:
#     red_values : [0, 10, 166, 255, 20, 255]
#     yellow_values : [25, 35, 17, 92, 74, 255]
#     green_values : [65, 90, 157, 255, 106, 255]

# Max Laptop
# color_detection:
#   ros__parameters:
#     red_values : [144, 179, 144, 255, 230, 255]
#     yellow_values : [0, 44, 53, 184, 222, 255]
#     green_values : [41, 73, 144, 255, 106, 255]

# PZB
color_detection:
  ros__parameters:
    red_values : [152, 179, 132, 187, 105, 255]
    yellow_values : [17, 36, 90, 190, 88, 165]
    green_values : [41, 73, 71, 253, 42, 146]